src/converters/fromZigbee.ts

```diff
@@ -740,8 +740,8 @@ const converters1 = {
         convert: (model, msg, publish, options, meta) => {
             if (utils.hasAlreadyProcessedMessage(msg, model)) return;
             const payload: KeyValueAny = {};
-            const multiplier = msg.endpoint.getClusterAttributeValue('seMetering', 'multiplier') as number;
-            const divisor = msg.endpoint.getClusterAttributeValue('seMetering', 'divisor') as number;
+            const multiplier = (msg.endpoint.getClusterAttributeValue('seMetering', 'multiplier') ?? 1) as number;
+            const divisor = (msg.endpoint.getClusterAttributeValue('seMetering', 'divisor') ?? 1000) as number;
```

Comment:
@Koenkk one thought here, could it be an idea to convert this to Number, instead of doing `as number`?

like this instead
```
const divisor = Number(msg.endpoint.getClusterAttributeValue('seMetering', 'divisor') ?? 1000);
```

Reply:
This value is always a number (or `undefined`) so it should not be needed.

Reply:
I had problems, because they where undefined, and so factor became null. Then on line 756, it checks if factor is not null, before calculating anything. That lead up to it reporting null as energy consumption.

When I added those default values, it was reporting like it should.

Reply:
The divisor/multiplier can be forced in the configure. I would propose to have the same logic as `instantaneousDemand` here, expose raw value when no factor is defined

Reply:
hmm.. even though that I defined the metering entity like 
```js
  electricityMeter({cluster: 'metering', power: {divisor: 1000, multiplier: 1}, energy: {divisor: 1000, multiplier: 1}}),
```
I had trouble in the fromZigbee.metering function. This was the previous release though, so things might have changed now.

Anyways, I have changed the logic a bit, so it now returns the raw value.. I'll get around to test things in a couple of days.